lochness/rpms/__init__.py

```diff
@@ -293,9 +294,24 @@ def initialize_metadata(Lochness: 'Lochness object',
 
                 # Consent date
                 if rpms_consent_colname in df_measure.index:
-                    subject_dict['Consent'] = datetime.strptime(
+                    consent_tmp = datetime.strptime(
                             df_measure[rpms_consent_colname],
-                            '%d/%m/%Y %I:%M:%S %p').strftime('%Y-%m-%d')
+                            '%d/%m/%Y %I:%M:%S %p')
```

Comment:
What is the value of `rpms_consent_colname` here? Is it provided via Lochness config file? @kcho 

I could trace it back to [here](https://github.com/AMP-SCZ/lochness/blob/e36b0d782f3291a4a5264cc49907e993f0811636/lochness/__init__.py#L95-L98). Still not clear but [this](https://github.com/AMP-SCZ/lochness/blob/11832e597ee6180237380ce18c677336eca1019d/tests/lochness_test/rpms/test_rpms.py#L319) says `chric_consent_date`.

Reply:
Yes, the value of `rpms_consent_colname` is set to `chric_consent_date` and it's loaded from configuration file. (In the beginning of the project, it wasn't clear what the variable name is, so `lochness.rpms` took this design.)